index.js

```diff
@@ -0,0 +1,56 @@
+module.exports = [
+  {
+    name: 'massively',
+    description: '...',
+    contentSources: {
+      plain: {
+        url: 'https://github.com/gatsbymanor/gatsby-theme-massively'
+      }
+    }
+  },
+  {
+    name: 'lens',
+    description: '...',
+    contentSources: {
+      plain: {
+        url: 'https://github.com/gatsbymanor/gatsby-lens'
```

Comment:
Typo: `theme` is missing from the name.

Should be `https://github.com/gatsbymanor/gatsby-theme-lens`